packages/dds/tree/src/index.ts

```diff
@@ -3,13 +3,33 @@
  * Licensed under the MIT License.
  */
 
-export { EmptyKey, FieldKey, TreeType, Value, TreeValue } from "./tree";
+export {
```

Comment:
Consider formatting these export statements more consistently. I personally like one item per line unless the entire statement fits on one line.

Reply:
Same, though I'm gonna wait until/if we have a formatter that cares about this before I care.